diff --git a/node_modules/@radix-ui/react-select/dist/index.d.mts b/node_modules/@radix-ui/react-select/dist/index.d.mts
index 5c7c730..5c01e6a 100644
--- a/node_modules/@radix-ui/react-select/dist/index.d.mts
+++ b/node_modules/@radix-ui/react-select/dist/index.d.mts
@@ -10,9 +10,9 @@ type Direction = 'ltr' | 'rtl';
 declare const createSelectScope: _radix_ui_react_context.CreateScope;
 interface SelectProps {
     children?: React.ReactNode;
-    value?: string;
-    defaultValue?: string;
-    onValueChange?(value: string): void;
+    value?: PropertyKey;
+    defaultValue?: PropertyKey;
+    onValueChange?(value: PropertyKey): void;
     open?: boolean;
     defaultOpen?: boolean;
     onOpenChange?(open: boolean): void;
@@ -88,7 +88,7 @@ interface SelectLabelProps extends PrimitiveDivProps {
 }
 declare const SelectLabel: React.ForwardRefExoticComponent<SelectLabelProps & React.RefAttributes<HTMLDivElement>>;
 interface SelectItemProps extends PrimitiveDivProps {
-    value: string;
+    value: PropertyKey;
     disabled?: boolean;
     textValue?: string;
 }
